ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml

```diff
@@ -10039,16 +10039,6 @@ periodics:
     job-release: "4.12"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-aws-ipi-p3-f28-long-duration-part3-3
-  reporter_config:
```

Comment:
Are those removal expected ?

Reply:
That's weird, I didn't make those changes, I'll revert it